db/migrate/20220512123911_create_request_logs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRequestLogs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :request_logs do |t|
+      t.references :miq_requests, :type => :bigint
+      t.string :log_message
```

Comment:
The `log_` prefix is redundant considering this is the request_logs table.  I think this can just be `message`.